chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,26 @@
+The women of Dressara often wore dresses, which Leanne both admired and found a shame. Leanne enjoyed wearing dresses, but found women in suits much more handsome.
+
+Leanne assumed the women of Dressara did something more than simply wear dresses and stroll about, but if they did, she never learned what it was. The men, she knew, did nothing more than stand about. They usually wore suits, and they were utterly boring, both as people and as targets. 
+
+Dressara was a somewhat pleasant place, if you enjoyed their sort of pomp, much unlike the rather dour planet Seven, around which Dressara held a lazy orbit. Like anywhere else, however, it was rather difficult to land on illegally.
```

Comment:
> held a typically lazy orbit—though recently the orbit was being adjusted 

I'd like to throw in something about orbital adjustment, and perhaps some mention of exploit (without explaining what it is yet). However, I'd also like to be a bit vague on the actual reasoning for the orbital adjustment; I'd like it to feel strange; uneasy, even.